src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,28 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
<img width="555" alt="image" src="https://github.com/user-attachments/assets/dd614698-53d7-4824-9a4a-106710a806c9">
user can be null, so better to add the check
```suggestion
        {post.user && <UserInfo user={post.user} />}
```